lib/std/hash/siphash.zig

```diff
@@ -111,9 +114,11 @@ fn SipHashStateless(comptime T: type, comptime c_rounds: usize, comptime d_round
             const m = mem.readIntSliceLittle(u64, b[0..]);
             self.v3 ^= m;
 
+            // TODO this is a workaround, should be able to supply the value without a separate variable
```

Comment:
Is there a related issue open for this TODO?

Reply:
Only this one :wink: #363